distributed/script/dserver.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Copyright (c) Orient Technologies LTD (http://www.orientechnologies.com)
 #
-./server.sh -Ddistributed=true $*
+"${0/%dserver.sh/server.sh}" -Ddistributed=true $*
```

Comment:
- paths with spaces were breaking this
- there might be other situations where this breaks